src/app-layer-htp-range.h

```diff
@@ -69,8 +69,8 @@ typedef struct HttpRangeContainerFile {
     uint16_t flags;
     /** wether a range file is currently appending */
     bool appending;
-    /** mutex */
-    SCMutex mutex;
+    /** error condition for this range. Its up to timeout handling to cleanup */
+    bool error;
```

Comment:
This is nice.
Should we update `expire` so that they get cleant faster ?

Reply:
Yeah that is a good idea.

Reply:
Actually I think its cleaner to add a check for `cu->error` in `ContainerValueRangeTimeout`. Trying that.